sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_directory_client.hpp

```diff
@@ -149,12 +149,13 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
      * entry (ACE) consists of a scope, a type, a user or group identifier, and permissions.
      * @param options Optional parameters to set an access control recursively to the resource the
      * directory points to.
-     * @return Azure::Core::Response<SetDirectoryAccessControlRecursiveResult>
+     * @return Azure::Core::Response<Models::SetDirectoryAccessControlRecursiveResult>
      * @remark This request is sent to dfs endpoint.
      */
-    Azure::Core::Response<SetDirectoryAccessControlRecursiveResult> SetAccessControlRecursive(
-        PathSetAccessControlRecursiveMode mode,
-        std::vector<Acl> acls,
+    Azure::Core::Response<Models::SetDirectoryAccessControlRecursiveResult>
+    SetAccessControlRecursive(
+        Models::PathSetAccessControlRecursiveMode mode,
+        std::vector<Models::Acl> acls,
```

Comment:
Are you sure you re-formatted with clang-format?

Reply:
Yes, can show you offline if you think is neccessary.